api/src/middleware/jwt-middleware.ts

```diff
@@ -0,0 +1,25 @@
+import { expressjwt as jwt } from "express-jwt"
+import jwksRsa, { type GetVerificationKey } from "jwks-rsa"
+
+import { AUTH0_DOMAIN, AUTH0_AUDIENCE, NODE_ENV } from "@/config"
+import logger from "@/utils/logger"
+
+if (NODE_ENV !== "test") {
+  logger.info(`AUTH0_DOMAIN=${AUTH0_DOMAIN}/.well-known/jwks.json`)
+}
```

Comment:
Now that we have a logger, and can customize the default log level, I think this can be rewritten as
```
logger.info(`AUTH0_DOMAIN=${AUTH0_DOMAIN}/.well-known/jwks.json`)
```
and it won't show up in the test logs.

It might also make sense to use `logger.debug` for these kinds of statements.